src/structures/ClientUser.js

```diff
@@ -230,9 +230,12 @@ class ClientUser extends Structures.get('User') {
    *   .catch(console.error);
    */
   setStatus(status) {
+    const statusArr = ['dnd', 'online', 'invisible', 'idle'];
```

Comment:
I'm not sure what made you decide this order but personally, I think the array should be in the same order it's displayed in Discord so `['online', 'idle', 'dnd', 'invisible']`.

Reply:
idk if it wouldnt be even better to just use the array in the if directly without putting it into a variable first since its not used anywhere else anyway. I agree on the order tho.

Reply:
This kind of thing definitely deserves to be in the Constants files and not just randomly in here.